payara-micro-gradle-plugin/build.gradle

```diff
@@ -54,3 +55,15 @@ pluginBundle {
         }
     }
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = "fish.payara.gradle.plugins"
+            artifactId = "payara-micro-gradle-plugin"
+            version = "1.0.4-SNAPSHOT"
```

Comment:
I assume this will get changed during release

Reply:
Yes, it will be changed, used to test the plugin locally by publishing to local maven.